packages/core/src/codewhisperer/service/transformByQ/transformationHubViewProvider.ts

```diff
@@ -203,25 +381,107 @@ export class TransformationHubViewProvider implements vscode.WebviewViewProvider
                         transform: rotate(360deg);
                     }
                 }
+                body {
+                    margin: 0;
+                    padding: 0 1em;
+                    height: 100vh;
+                }
+
+                .wrapper {
+                    height: 100%;
+                    display: flex;
+                }
+
                 .spinner {
                     display: inline-block;
                     animation: spin 1s infinite;
                 }
+
+                .column--container, .substep-container {
+                    display: flex;
+                    flex-direction: row;
+                }
+
+                .column {
+                    flex-grow: 1;
+                }
+
+                .substep-container  p {
+                    margin: .5em 0;
+                }
+
+                .step, .simple-step {
+                    padding: .5em 0 .5em 0;
+                    margin: 0;
+                }
+
+                .step {
+                    padding-left: 20px;
+                }
+
+                .step:hover, .active {
+                    background-color: aliceblue;
+                    background-color: var(button.hoverBackground);
+                }
+
+                #stepdetails {
```

Comment:
Discussion: To get rid of the border on a certain state you can:
1. remove this element by id #stepdetails when you are done
2. Add an attribute to the element #stepdetails and target that in css eg)
```
#stepdetails [data-status="COMPLETED"] {
   border: none;
   // or 
   border-left: 1px solid transparent
}
<div data-status="COMPLETED" id="stepdetails">...<div>
```